packages/webpack-plugin/src/index.js

```diff
@@ -123,6 +123,25 @@ class LoadablePlugin {
             },
           )
         })
+
```

Comment:
checked - SriPlugin extends assets information at PROCESS_ASSETS_STAGE_OPTIMIZE_HASH stage, so at PROCESS_ASSETS_STAGE_REPORT all extra fields should be already a part of stats.
- https://webpack.js.org/api/compilation-hooks/
- https://github.com/waysact/webpack-subresource-integrity/blob/master/index.js#L396

Reply:
youre right, thanks! i wasnt seeing the integrity available at this stage before but i must have been looking in the wrong place at the time